node/libs/crypto/src/bn254/mod.rs

```diff
@@ -0,0 +1,200 @@
+//! BLS signature scheme for BN254 curve.
+
+use std::collections::BTreeMap;
+
+use anyhow::anyhow;
+use bn254::{PrivateKey, ECDSA};
+use rand::Rng;
+
+pub use crate::bn254::error::Error;
+use crate::ByteFmt;
+
+mod error;
+
+mod testonly;
+#[cfg(test)]
+mod tests;
+
+pub struct SecretKey(PrivateKey);
+
+impl SecretKey {
+    /// Generates a random secret key
+    pub fn random<R: Rng>(rng: &mut R) -> Self {
+        let private_key = PrivateKey::random(rng);
+        return Self(private_key);
+    }
```

Comment:
implement "Distribution<SecretKey> for Standard" instead.

Reply:
Fixed.